packages/ui5-application-writer/src/index.ts

```diff
@@ -9,7 +9,8 @@ import { mergeWithDefaults } from './data';
 import { ui5TSSupport } from './data/ui5Libs';
 import { applyOptionalFeatures, enableTypescript as enableTypescriptOption } from './options';
 import { Ui5App } from './types';
-import { getEsmTypesVersion } from './data/defaults';
+import { getEsmTypesVersion, mergeObjects } from './data/defaults';
+import { getFilePaths } from './files';
```

Comment:
Making these part of the public interface of the write module so that you can reuse them in the new write is a bad design because
* the writers shouldn't depend on each other
* these functions do not belong into a writer API

Suggestions to solve it:
* `getFilePaths` looks like a function that fits into the `project-access` module, so maybe move it there and make both writers depend on it
* `getEsmTypesVersion` and `mergeObjects` (and even more in the `./data/defaults` file are generic UI5 configuration utility functions, so it might make sense to move them to the `ui5-config` module

@devinea what do you think?
@cianmSAP if both Austin and you agree, could you please do that in a separate PR, so that this one only contains the changes for your new module

Reply:
The comment was supposed to be in line 133 ;)

Reply:
Thanks for the explanation @tobiasqueck , it makes sense. 

I've created a [TBI](https://github.com/SAP/open-ux-tools/issues/1039) and [draft PR](https://github.com/SAP/open-ux-tools/pull/1040) in if all are agreed.